include/deal.II/base/tensor.h

```diff
@@ -227,6 +233,12 @@ class Tensor<0, dim, Number>
    */
   constexpr DEAL_II_CUDA_HOST_DEV operator const Number &() const;
 
+  /**
+   * Copy the Tensor into a std::array.
```

Comment:
```suggestion
   * Copy the elements of the Tensor into a std::array.
```
Do you want to say that the order is that defined by the `*unroll*index/indices` functions?